src/scss/skin-modern/components/_subtitleoverlay.scss

```diff
@@ -6,14 +6,27 @@
 
   @include hidden;
 
-  bottom: 2em;
+  bottom: 0;
   font-size: 1.2em;
-  left: 3em;
+  left: 0;
   pointer-events: none;
   position: absolute;
-  right: 3em;
+  right: 0;
   text-align: center;
   transition: bottom $animation-duration-short ease-out;
+  top: 0;
+
+  &.#{$prefix}-default-subtitle-position {
```

Comment:
Please rename to `subtitle-position-default` as discussed in the review https://github.com/bitmovin/bitmovin-player-ui/pull/258#discussion_r325537968.